addons/stat/initSettings.sqf

```diff
@@ -9,3 +9,41 @@
     {},
     true
 ] call CBA_fnc_addSetting;
+
+// Log to RPT
+[
+    QGVAR(DebugRPT),
+    "CHECKBOX",
+    ["Log Stats To RPT","Print Statistics to RPT. May cause extra stress."],
```

Comment:
Pretty sure writing to RPT is now async from the main process so it shouldn't slow it down.
Somewhat relevant discord quote:
> What? logging has been async since 2.00 I think

[Dedmen](https://discord.com/channels/105462288051380224/105465701543723008/828671477788311602)